eva/optimizer/plan_generator.py

```diff
@@ -49,13 +49,15 @@ def build_optimal_physical_plan(
         physical_plan = None
         root_grp = optimizer_context.memo.groups[root_grp_id]
         best_grp_expr = root_grp.get_best_expr(PropertyType.DEFAULT)
-        physical_plan = best_grp_expr.opr
 
-        for child_grp_id in best_grp_expr.children:
-            child_plan = self.build_optimal_physical_plan(
-                child_grp_id, optimizer_context
-            )
-            physical_plan.append_child(child_plan)
+        if best_grp_expr is not None:
```

Comment:
For my education, in which case, the `best_grp_expr` can be None?

Reply:
It shouldn't happen if plans are implemented correctly. Will clean it up in the next commit.